### DISCUSSION TITLE
Auto page refresh on the kiosk browser

### DISCUSSION THREAD

**Discussion by Webbyfrost2** [guysoft/FullPageOS#596] [Q&A]
Hey everyone,

Apologies if this is something that has been asked and answered before. I have tried searching for similar discussions and through the closed/opened issues pages and couldn't see an answer. I am also a noob with pi configurations and slowly teaching myself the basics. 

I am using the fullpageos to display a dashboard that is on 1 webpage and it is working really well. The only thing is that it sometimes stop getting the new data and the last data received time clock on the page sops being updated. I think all i need is a simple page refresher and I found the ctrl + t command to open a new window in non kiosk mode and had a wide look at browser extensions and all the ones I found required a human to start/stop the extension. 

Am I missing something pretty basic here? 

Its an amazing product and I wish I had the funds at the moment to donate. I do really appreciate you putting this out there for people to use. 

Thanks for being awesome.

Helen

**Comment by guysoft** [comment]
There is [a manual refresh script](https://github.com/guysoft/FullPageOS/blob/devel/src/modules/fullpageos/filesystem/opt/custompios/scripts/refresh)

Also there is also a script that refreshes on error. so if your page gives an error this would refresh when an error is detected. you need to comment two lines here, the file on the os would be in ``/opt/custompios/scripts/start_chromium_browser`` or in ``/home/pi/scripts/start_chromium_browser`` depending on your version:

https://github.com/guysoft/FullPageOS/blob/devel/src/modules/fullpageos/filesystem/opt/custompios/scripts/start_chromium_browser#L20
